google/cloud/internal/rest_stub_helpers.h

```diff
@@ -36,6 +36,10 @@ Status RestResponseToProto(google::protobuf::Message& destination,
 Status ProtoRequestToJsonPayload(google::protobuf::Message const& request,
                                  std::string& json_payload);
 
+rest_internal::RestRequest CreateRestRequest(
+    std::string path,
+    std::vector<std::pair<std::string, std::string>> query_params = {});
```

Comment:
super nit: bww will tell you not to default params for internal functions. Especially when the value is always supplied.

optional: define the thing in the .cc only given that we are not testing it? Unless you have plans to use it elsewhere.

Reply:
Don't default params for internal functions. Especially when the value is always supplied.